src/bin/pgcopydb/parsing_utils.c

```diff
@@ -631,7 +631,7 @@ parse_pguri_info_key_vals(const char *pguri,
 		{
 			if (streq(keyword, option->keyword))
 			{
-				found = option->val != NULL && !streq(option->val, "");
+				found = !streq(option->val, "");
```

Comment:
Why can't `option->val` be NULL here? can we add a comment about that?

Reply:
The reason for the change is that `streq` already has the null checks by definition.